build.sbt

```diff
@@ -628,6 +630,34 @@ lazy val sbtProj = (project in file("sbt"))
   )
   .configure(addSbtCompilerBridge)
 
+lazy val sbtBig = (project in file(".big"))
+  .dependsOn(sbtProj)
+  .settings(
+    name := "sbt-big",
+    normalizedName := "sbt-big",
+    assemblyShadeRules.in(assembly) := Seq(
+      ShadeRule.rename("fastparse.**" -> "sbt.internal.fastparse.@1").inAll
```

Comment:
There are more external packages that should be shaded, here is the full list by looking at all classfiles for `org.scala-sbt:sbt:1.2.6`

```
com/
fastparse/
gigahorse/
google/
jawn/
jline/
native/
okhttp3/
okio/
org/
sbinary/
sbt/
scala/
scalacache/
scalapb/
sjsonnew/
sourcecode/
xsbt/
xsbti/
```